ds3-sdk/src/main/java/com/spectralogic/ds3client/models/bulk/Ds3Object.java

```diff
@@ -126,6 +127,6 @@ public boolean equals(final Object obj) {
         final Ds3Object ds3Obj = (Ds3Object) obj;
         return ds3Obj.getName().equals(this.getName()) &&
```

Comment:
If you're in here updating the way versionIds are checked for equality, you should do the same for the name as well.

Reply:
Ooh I didn't realize that name was nullable, I'll fix it